src/pages/studyView/queryStudyAnalytics.ts

```diff
@@ -0,0 +1,24 @@
+import * as request from "superagent";
+
+export interface IAnalyticsResponse {
```

Comment:
Lets call it PortalAnalyticsResponse

Reply:
Actually, lets just call it PortalAnalyticsData since the fact that was delivered via api is immaterial

Reply:
So renamed